### ISSUE TITLE
Support Let's Encrypt SSL certificates for S3 Backup endpoints

### ISSUE THREAD

**Issue by DenisKramer** [longhorn/longhorn#696]
## Issue
When trying to use a S3 backup store using Let's Encrypt certificates (MinIO) via https, access fails with 

```
x509: certificate signed by unknown authority
```

errors.

## Workaround
 Manually adding the [Let's Encrypt Intermediate X3 certificate](https://letsencrypt.org/certificates/) to /etc/ssl/certs/ca-certificates/ca-certificates.crt in the various containers helps avoiding this error. 

## Proposed solution
Add Let's Encrypt Intermediate certificates to container images.

**Comment by sybnex** [comment]
Im using MinIO with a subdomain and lets encrypt certificates for **production** and I never got an error message like this.

So why you are using Intermediate and not Production? ...

**Comment by DenisKramer** [comment]
Thanks. I am using **production** certificates as well. As it is obviously working for you, I was digging a bit deeper. I used an older version of MinIO (mid 2018) that could only deal with the leaf certificate (cert.pem generated by certbot). That version of MinIO crashed when trying to use the fullchain.pem that includes the intermediate. After updating to the latest version, I was able to use the fullchain.pem as the public certificate in MinIO. MinIO now sends the intermediate certificate along to enable verification against the root certificate, which solves the issue.

I guess, it might be helpful to have a sentence in the docs making clear that one should use the fullchain.pem (as advised in the current [MinIO docs](https://docs.min.io/docs/generate-let-s-encypt-certificate-using-concert-for-minio.html) to help others with legacy MinIO setups.
